### ISSUE TITLE
A11y_AzureDataStudio_AddNewController_ScreenReader : NVDA is not announcing the header word for "Add new controller". 

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7950]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;DesktopApp,#NVDA;#JAWS;#SQL Azure Data Studio;#Win32

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : NVDA , JAWS

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Connect to the servers with the required credentials.
3. Navigate to the "SQL Server Big Data Cluster" and expand it.
4. Navigate to the "SQL Server Big Data Cluster controller" and click on it.
5. Start NVDA.
6. Navigate till "Add new Controller "  using arrow navigation.
7. Observe the announcement by screen reader.

**Actual:**
NVDA is not announcing the header word for "Add new controller".
It only announces that "Add new controller" .
**Similar issue is also observed with JAWS .**

**Expected:**
Screen reader should announce header word for the "Add new controller" .

**User Impact:**
Screen reader user won't able to differentiate between the content and the header .

**Attachment for Reference:**
![7950_A11y_AzureDataStudio_AddNewController_ScreenReader](https://user-images.githubusercontent.com/54656883/67397052-40bd7a80-f5c6-11e9-8a24-31cdd47322be.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![7950_Closed](https://user-images.githubusercontent.com/54656883/72724812-597c0300-3baa-11ea-8559-72dc7e9d3f49.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
